### PR TITLE
fix: fixes on deploy. add edition to published items

### PR DESCRIPTION
**Pull request by nicosantangelo**

### CODE REVIEW

File: src/components/Modals/MintItemsModal/MintItemsModal.tsx [hunk 1]
```diff
@@ -103,8 +107,8 @@ export default class MintItemsModal extends React.PureComponent<Props, State> {
                   {t('global.cancel')}
                 </Button>
               ) : (
-                <Button primary onClick={this.handleMintItems} loading={isLoading}>
-                  {t('global.done')}
+                <Button primary onClick={this.handleMintItems} loading={isLoading} disabled={this.isDisabled()}>
```
Comment (nachomazzara) [inline src/components/Modals/MintItemsModal/MintItemsModal.tsx:110]: Maybe we can move `this.isDisabled()` before the render's _return_ to just use variables for components

File: src/modules/item/export.ts [hunk 2]
```diff
@@ -41,9 +42,9 @@ async function getFiles(contents: Record<string, string>) {
   }, {})
 }
 
-function toCatalystItem(collection: Collection, item: Item): CatalystItem {
+function toCatalystItem(collection: Collection, item: Item, chainId: ChainId): CatalystItem {
   return {
-    id: item.id,
+    id: getCatalystURN(collection, item, chainId),
```
Comment (nachomazzara) [inline src/modules/item/export.ts:47]: I know that we have only collections right now, but maybe we can call it `getCatalystCollectionItemURN` or `getCatalystItemURN`. What do you think?

File: src/modules/translation/languages/en.json [hunk 3]
```diff
@@ -951,7 +951,8 @@
   "mint_items_modal": {
     "title": "Mint Items",
     "add_item": "Add Item",
-    "no_items": "You have no items available to mint for \"{name}\""
+    "no_items": "You have no items available to mint for \"{name}\"",
+    "mint": "Mint"
```
Comment (nachomazzara) [inline src/modules/translation/languages/en.json:955]: What do you think of moving `mint` to global?

**Review by nachomazzara — COMMENTED** [review]
🚀  🧹

File: src/modules/item/export.ts (hunk 2 above)
Comment (nicosantangelo) [reply src/modules/item/export.ts:47]: Oh good point, didn't realize it was scoped by collection

File: src/modules/translation/languages/en.json (hunk 3 above)
Comment (nicosantangelo) [reply src/modules/translation/languages/en.json:955]: will do

**Pull request merged**